articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -0,0 +1,254 @@
+- We set the `audience` to `https://${account.namespace}/api/v2/`
+- We asked for the additional scope `read:current_user`
+
+Once you have the Access Token you can use it to call the endpoint. This part remains the same, nothing else changes in the request except for the value you use as `Bearer` token. The response remains also the same.
+
+```har
+{
+  "method": "GET",
+  "url": "https://${account.namespace}/api/v2/users/USER_ID",
+  "headers": [{
+    "name": "Authorization",
+    "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+  }]
+}
+```
+
+### Use cross-origin authentication
+
+(to be updated)
+
+## Changes in Account Linking
+
+We are introducing some changes in the Account Linking process, one of which is breaking.
+
+
+You can no longer invoke the endpoint with the authenticated primary account's ID Token in the `Authorization` header. You have to use an Access Token which must include the `update:current_user_identities` scope. We have already seen how you can get an Access Token. In the following snippets you can see what changes in the invocation of the endpoint (that is, only the value you use in the `Authorization` header).
+
+<div class="code-picker">
+  <div class="languages-bar">
+    <ul>
+      <li class="active"><a href="#with-id-token" data-toggle="tab">Legacy (ID Token)</a></li>
+      <li><a href="#with-access-token" data-toggle="tab">Current (Access Token)</a></li>
+    </ul>
+  </div>
+  <div class="tab-content">
+    <div id="with-id-token" class="tab-pane active">
+      <pre class="text hljs">
+        <code>
+POST https://${account.namespace}/api/v2/users/PRIMARY_ACCOUNT_USER_ID/identities
+Authorization: "Bearer PRIMARY_ACCOUNT_ID_TOKEN"
+Content-Type: application/json
+{
+  "link_with": "SECONDARY_ACCOUNT_ID_TOKEN"
+}
+        </code>
+      </pre>
+    </div>
+    <div id="with-access-token" class="tab-pane">
+      <pre class="text hljs">
+        <code>
+POST https://${account.namespace}/api/v2/users/PRIMARY_ACCOUNT_USER_ID/identities
+Authorization: "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+Content-Type: application/json
+{
+  "link_with": "SECONDARY_ACCOUNT_ID_TOKEN"
+}
+        </code>
+      </pre>
+    </div>
+  </div>
+</div>
+
+In order to use this new implementation, and send the `SECONDARY_ACCOUNT_ID_TOKEN` as part of the payload, the following must apply:
+- The ID Token must be signed using `RS256` (you can set this value at *Dashboard > Clients > Client Settings > Advanced Settings > OAuth*)
+- The claim `aud` of the ID Token, must identify the client, and be the same value with the `azp` claim of the Access Token
+
+If you already use Access Tokens at the `Authorization` header (use case described at the second bullet of [Linking User Accounts](/link-accounts#the-management-api)) you don't need to change something in your implementation, but do note that now you can send at the request's body the secondary account's ID Token, instead of the `user_id` (or you can keep using the `user_id`).
```

Comment:
This gives the impression that I can choose between `user_id` and token. In reality, only token is supported with `current_user` scope.